zng/docs/spec.md

```diff
@@ -133,8 +133,8 @@ needed to define the schema for each distinct record.  To define a new
 type, the "#" syntax is used.  For example,
 logs from the open-source Zeek system might look like this
 ```
-#24:record[_path:string,ts:time,uid:string,id:record[orig_h:ip,orig_p:port,resp_h:ip,resp_p:port]...
-#25:record[_path:string,ts:time,fuid:string,tx_hosts:set[ip]...
+#24:record[_path:string,ts:time,uid:bstring,id:record[orig_h:ip,orig_p:port,resp_h:ip,resp_p:port]...
+#25:record[_path:string,ts:time,fuid:bstring,tx_hosts:set[ip]...
```

Comment:
This adjustment was just for correctness, since for some time we've been representing every Zeek `string` field other than `_path` as a `bstring`.